lib/livebook_cli/server.ex

```diff
@@ -127,6 +127,12 @@ defmodule LivebookCLI.Server do
       |> Livebook.Utils.browser_open()
     end
   end
+  
+  defp open_from_options(base_url, opts, [path])
+    path = Path.expand(path)
+    url = LivebookWeb.Helpers.notebook_import_url("file://" <> path)
+    Livebook.Utils.browser_open(url)
+  end
```

Comment:
Yeah, we still need a catch-all that calls `print_error` saying it expects at most one argument. :)